src/lib/indexer/index.ts

```diff
@@ -254,4 +278,52 @@ export default class Indexer {
       }
     })
   }
+  getStatus = async (): Promise<Status> => {
+    const indexerTimeout = this.env.INDEXER_TIMEOUT_MS
+    const currentDate = new Date()
+    if (currentDate.getTime() - this.startupTime.getTime() < indexerTimeout) {
+      // if we started less than 30s ago -> PASS
+      return {
+        status: serviceState.UP,
+        detail: {
+          message: 'Service healthy. Starting up.',
+          startupTime: this.startupTime,
+          latestActivityTime: currentDate,
+        },
+      }
+    }
+    const latestActivityTime = this.lastProcessedBlockTime || this.lastUnprocessedBlockTime
+    if (this.lastProcessedBlockTime === null && this.lastUnprocessedBlockTime === null) {
```

Comment:
this lint can be `if (latestActivityTime === null)`

Reply:
I don't think it can be...?
because this say if both are null at this point
`if (this.lastProcessedBlockTime === null && this.lastUnprocessedBlockTime === null)`
but this one says if either one of these is null at this point
`const latestActivityTime = this.lastProcessedBlockTime || this.lastUnprocessedBlockTime`